examples/multiplatform/watchOSAppExtension/Tests/UnitTests/BUILD

```
@@ -2,20 +2,20 @@ load("@build_bazel_rules_apple//apple:watchos.bzl", "watchos_unit_test")
 load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
 
 watchos_unit_test(
-    name = "ExampleWatchKitAppTests",
+    name = "UnitTests",
     minimum_os_version = "7.0",
     tags = ["manual"],
     visibility = ["//visibility:public"],
-    deps = [":ExampleWatchKitAppTests.library"],
+    deps = [":UnitTests.library"],
 )
 
 swift_library(
-    name = "ExampleWatchKitAppTests.library",
+    name = "UnitTests.library",
     testonly = True,
     srcs = glob(["**/*.swift"]),
-    module_name = "ExampleWatchKitAppTests",
+    module_name = "UnitTestsLibrary",
     tags = ["manual"],
     deps = [
-        "//examples/watchos_app/ExampleWatchKitExtension:ExampleWatchKitExtension.library",
+        "//examples/multiplatform/watchOSAppExtension",
```

```suggestion
        "//examples/multiplatform/watchOSAppExtension:watchOSAppExtension.library",
```

This is needed to fix the multiple configuration issues.